### ISSUE TITLE
move from rackspace / wheelhouse-uploader to anaconda.org / ananconda

### ISSUE THREAD

**Issue by mattip** [MacPython/astropy-wheels#66]
The hosting on rackspace of http://wheels.scipy.org is no more. Other MacPython projects have moved to using https://anaconda.org/scipy-wheels-nightly/ for weekly uploads and https://anaconda.org/multibuild-wheels-staging for staging wheels to PyPI. For instance, here is the numpy code (on travis) to do this https://github.com/MacPython/numpy-wheels/blob/master/.travis.yml#L99

You have access to tokens via the "Organization Secrets" https://github.com/MacPython/astropy-wheels/settings/secrets.

xref @matthew-brett @astrofrog

**Comment by astrofrog** [comment]
@mattip - the example you showed in numpy-wheels seems to use a Travis secret not a GitHub org secret. Does this mean we have to switch to GitHub actions, or is there a way for Travis to access the secrets? And how should we deal with AppVeyor?

**Comment by mattip** [comment]
No, that is for github actions only. For each other CI you need to create specific secrets. I don't know who owns https://travis-ci.org/github/MacPython/astropy-wheels/ - whoever it is can see the "settings" option under the "more options" hamburger menu on the right and get to the secrets section. If whoever it is cannot already create a token on anaconda.org, I could add them. Or one of the people who can create tokens on anaconda.org could take over the CI on https://travis-ci.org/github/MacPython/astropy-wheels/.

Similarly for AppVeyor.

Alternatively, you could try moving to github actions. If you wish to release wheels for ARM64, the best option (in my opinion) is still travis, since github actions only runs x86.

**Comment by saimn** [comment]
Hi @mattip, could you please add me to the organization on anaconda.org so I can create the tokens ? 
My username is "saimon" on anaconda.org

**Comment by mattip** [comment]
Could one of the core maintainers vouch for @saimn? I don't see anyone by that name as a committer here.

**Comment by astrofrog** [comment]
@mattip I vouch for @saimn!

**Comment by astrofrog** [comment]
I added him as an admin to this repo

**Comment by mattip** [comment]
Thanks. Done.

**Comment by saimn** [comment]
Thanks @mattip !

**Comment by astrofrog** [comment]
@mattip - would it be possible to add myself and @saimn to the AppVeyor project for astropy-wheels?

**Comment by mattip** [comment]
I added @saimn. Is there a problem creating the tokens?

**Comment by saimn** [comment]
@mattip - I could create the tokens on anaconda.org, and add them in Travis, but I don't have access to AppVeyor's admin interface. Do you know if something must be done to get this access ?

**Comment by astrofrog** [comment]
(I also can't access the AppVeyor admin interface)

**Comment by mattip** [comment]
I think @matthew-brett might know?

**Comment by mattip** [comment]
(another reason to move to github actions for all x86 builds)
